CMakeLists.txt

```diff
@@ -79,6 +81,16 @@ add_real_time_tools_demo(demo_checkpoint_timer)
 #add_executable(${EXE2} src/bin/realtime_test_display.cpp)
 #target_link_libraries(${EXE2} ${LIBRARY_NAME} ${catkin_LIBRARIES})
 
+##################
+# Python wrapper #
+##################
+
+pybind11_add_module(real_time_tools_py srcpy/wrappers.cpp)
```

Comment:
Naive question: is the \`project name\`_py the convention we use?

Reply:
no, it is not. It should be real_time_tools, but then the name conflicts with the main library of the package. There should be some trick to allow both libraries to have the same name, but I could never get it to work, and gave up --;

Reply:
It is not too difficult once you get the trick.

```
################################
# python bindings with pybind11#
################################
pybind11_add_module(py_${PROJECT_NAME}
    srcpy/bindings1.cpp
    srcpy/bindings2.cpp
)
target_link_libraries(
    py_${PROJECT_NAME} PRIVATE ${PROJECT_NAME} ${catkin_LIBRARIES})
set_target_properties(py_${PROJECT_NAME} PROPERTIES
  LIBRARY_OUTPUT_DIRECTORY ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_PYTHON_DESTINATION}
  OUTPUT_NAME ${PROJECT_NAME}
)
install(TARGETS py_${PROJECT_NAME} DESTINATION ${CATKIN_GLOBAL_PYTHON_DESTINATION})
```

Reply:
I will add this in the mpi_cmake_modules documentation.